scripts/caput-pipeline

```diff
@@ -121,5 +158,5 @@ mpirun -np %(mpiproc)i -npernode %(pernode)i -bind-to none python %(scriptpath)s
         os.system('cd %s; qsub jobscript.sh' % pbsdir)
 
 
-if __name__ == "__main__":
```

Comment:
You really don't like double quotes, eh?

Reply:
It's the inconsistency which was getting to me.

Reply:
Yeah. I can get behind that. I must have written this code in the dim and distant past when I was a double quotes kind of guy. I switched at somepoint, but now I'm writing a lot more C++ it's getting confusing again! :)